helm/README.md

```diff
@@ -0,0 +1,125 @@
+**Note**:  
+For minikube, Specs used to test the charts are. 3GB RAM and 40GB memory.
```

Comment:
Minimum is 4GB RAM. These notes are not necessary so please remove them.

Reply:
Noted